explore/src/main/scala/explore/targets/TargetSummaryTable.scala

```diff
@@ -241,11 +245,13 @@ object TargetSummaryTable extends TableHooks:
               )
             )
           ),
-          PrimeTable(
+          PrimeVirtualizedTable(
             table,
+            _ => 32.toPx,
             striped = true,
             compact = Compact.Very,
             tableMod = ExploreStyles.ExploreTable |+| ExploreStyles.ExploreSelectableTable,
+            containerMod = ExploreStyles.ExploreFullHeightTable,
```

Comment:
Did you try `AutoHeightPrimeVirtualizedTable`? Maybe this class is not necessary.

Reply:
Ah I could try that, I thought it was auto row height

Reply:
Yeah that works, thanks